frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -913,7 +921,7 @@ export const dashboardLogic = kea<dashboardLogicType>({
             actions.resetInterval()
             actions.refreshAllDashboardItems({ action: 'refresh_manual' })
         },
-        refreshAllDashboardItems: async ({ tiles, action, initialLoad, dashboardQueryId }, breakpoint) => {
+        refreshAllDashboardItems: async ({ tiles, action, initialLoad, dashboardQueryId = uuid() }, breakpoint) => {
```

Comment:
Is the = evaluated as in python (only once) or as you'd expect?

Reply:
Fair to check 😅 

But, yep, not like Python https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#evaluated_at_call_time